markdown/events/2021/bytesize-2-configs.md

```diff
@@ -34,3 +34,119 @@ This week, Maxime Garcia ([@MaxUlysse](http://github.com/MaxUlysse/)) will prese
 The talk will be presented on Zoom and live-streamed on YouTube:
 
 * YouTube: <https://youtu.be/cXBYusdjrc0>
+
+<details markdown="1"><summary>Video transcription</summary>
+**Note: This text has been edited to make it more suitable for readers.**
+
+OK, so hello everyone, Maxime here.
+I am working for the Swedish Childhood Tumor Biobank which is located at Karolinska Institutet and I am sitting half-time at the National Genomics Infrastructure at SciLifeLab.
+Today for the second nf-core Bytesize, I am going to talk about nf-core configs and how they work.
+
+What I like about Nextflow or in fact what I actually love about Nextflow, is its portability, shareability and reusability.
+As a challenge I am going to run [nf-core/eager](https://nf-co.re/eager) - a pipeline I am not very familiar with - on the provided test data.
+I will be assuming I am running it on a new machine with just Docker installed and I am first going to specify everything on the command-line without using any config file or any profile.
+This is not something you should normally attempt.
+
+[1:04]
```

Comment:
```suggestion
[[1:04](https://www.youtube.com/watch?v=cXBYusdjrc0&t=1m4s)]
```
The timestamps are very nice. I think they would be even more useful with a link to that time in the video.

Reply:
I like the idea of the timestamps with links to the video

Reply:
Oooh, advanced! Ok, I can fix that.